app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration-for-testing.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Work history iteration
+description: Moving the work history within Safeguarding section
+date: 2021-01-08
+related:
+  items:
+  - text: Work history breaks
+    href: https://bat-design-history.netlify.app/apply-for-teacher-training/work-history-breaks/
+    description: Previous post about breaks in work history
+  - text: Work history - August iteration
+    href: https://bat-design-history.netlify.app/apply-for-teacher-training/work-history/
+    description: Previous change to the work history flow
+---
+Discussions with providers revealed that many of them require a “full” work history, rather than just work within the past 5 years, for safeguarding reasons.
```

Comment:
Many providers told us that they want to see a candidate's full work history, not just the last 5 years - for safeguarding reasons.